### PR TITLE
Dockerize Bramble

### PR DESCRIPTION
**Pull request by gideonthomas**
This PR contains two main parts:
- Cleanup the Gruntfile and remove tasks that we aren't using (and probably never will). This is the first commit.
- Dockerize Brackets and add scripts to run it. These are the rest of the commits.

I keep the whitespace changes in because they keep popping up. So future PRs will not have them anymore. If you want to ignore them during review,  add `?w=1` to the url.

### CODE REVIEW

**Comment by humphd** [comment]
I've started reviewing this, I'll finish it in the next day or so.  Looking good so far.

**Comment by gideonthomas** [comment]
Note: I haven't tested the Windows script. Will do that soon.

File: Gruntfile.js [hunk 1]
```diff
@@ -348,49 +340,6 @@ module.exports = function (grunt) {
                 tasks: ['jshint:test']
             }
         },
-        /* FIXME (jasonsanjose): how to handle extension tests */
```
Comment (humphd) [inline Gruntfile.js]: I wonder if we should delete `test/*` since we aren't running them ever?

Comment (gideonthomas) [inline Gruntfile.js]: Yeah probably a good idea! I'll do that!

**Pull request closed without merging**